script/index.ts

```diff
@@ -55,14 +55,29 @@ function buildOptionsForLanguage(language: Language) {
       items: option.items
     };
   })
+  languageOptions["Beautifiers"] = {
+    title: "Beautifiers",
+    type: "array",
+    default: beautifiers,
+    items: {
+      type: "string"
```

Comment:
Could this be an enum from `beautifiers`? Basically, `beautifiers` option is an array of beautifier names, in order, and can pick any of the beautifiers names (not all `string`s).

Reply:
Not possible with Atom's settings UI unfortunately.